community/modules/compute/schedmd-slurm-gcp-v5-node-group/variables.tf

```diff
@@ -360,3 +360,9 @@ variable "access_config" {
   }))
   default = []
 }
+
+variable "disable_public_ips" {
```

Comment:
I am leaning towards making this, `enable_public_ip`. We have a task to deprecate all "disable" style variables, and in this case there is not even an underlying variable that we are trying to stay consistent with. I recognize that this creates an odd temporary situation where the variables are different on different components, so I could be convinced otherwise.

Reply:
I'd rather have consistency between modules rather than fixing the variable name in one module. 

I see two options that I'd be okay with:
1. Push this PR through now as is and make the breaking change for all modules later
1. Make the breaking change for all of them now, perhaps with backward compatibility built in.

I'd prefer (1) so that we can discuss and get agreement from everyone after the new year. Let me know your thoughts.

Reply:
OK 1 is fine.